src/containers/Main/AuthenticatedWrapper.tsx

```diff
@@ -7,7 +7,7 @@ import { Redirect } from 'react-router'
 
 const AuthenticatedContent: React.FC = () => {
   return isLoggedIn() ? (
-    <Container>
+    <Container fluid>
```

Comment:
This makes the app go full-width which looks pretty bad. If we need to use it to be responsive, then maybe add a `maxWidth` for now? We need to have a plan about implementing layout/styles at some point, but for now we should probably keep it like it is for the most part.

Just looked up `Container` in Semantic: "A container limits content to a maximum width."
And then also: "fluid |   | {bool} | Container has no maximum width.'

If all the `fluid` prop is doing is turning off maximum width, why do we need it?

Reply:
Good point... I'm removing it now, it doesn't seen to have improved at all.